website/docs/cloud-docs/api-docs/policies.mdx

```diff
@@ -70,17 +70,17 @@ This POST endpoint requires a JSON object with the following properties as a req
+| --------------------------------------- | -------------- | --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `data.type`                             | string         |                                               | Must be `"policies"`.                                                                                                                                                                                                                                                                                                                                                                                                    |
+| `data.attributes.name`                  | string         |                                               | The name of the policy, which can include letters, numbers, `-`, and `_`. You cannot modify this name after creation.                                                                                                                                                                                                                                                                                                    |
+| `data.attributes.description`           | string         | `null`                                        | Text describing the policy's purpose. This field supports Markdown and appears in the Terraform Cloud UI.                                                                                                                                                                                                                                                                                                                |
+| `data.attributes.kind`                  | string         | `sentinel`                                    | The policy-as-code framework for the policy. Valid values are `sentinel` and `opa`.                                                                                                                                                                                                                                                                                                                                      |
```

Comment:
one note that I just saw is that for "policies" we put it in double quotes, but here the values are just in code format. We should be consistent - unless is there a specific reason that policies is in those quotes...let me know if there is! just checking :-)